docs/guides/permissions.md

```diff
@@ -4,4 +4,66 @@ title: Permissions
 
 # Permissions
 
-Documentation coming soon
+Permissions can be managed using `Permission` classes. A `Permission` class extends `BasePermission` and has a `has_permission` method. It can be added to a field using the `permission_classes` keyword argument. A simple example looks like this:
+
+```python
+import strawberry
+from strawberry.permission import BasePermission
+from strawberry.types import Info
+
+class IsAuthenticated(BasePermission):
+    message = "User is not authenticated"
+
+    def has_permission(self, source: Any, info: Info, **kwargs) -> bool:
+        return False
+
+@strawberry.type
+class Query:
+    user: str = strawberry.field(permission_classes=[IsAuthenticated])
+```
+
+If the `has_permission` method fails then an error will be raised using the `message` class attribute. See [Dealing with Errors](/docs/guides/errors) for more information on how errors are handled.
```

Comment:
Might want to be explicit on what `fails` means. Returns false? Throws an exception?

Reply:
Have cleared this up as `False` or falsy.